src/main/java/org/broadinstitute/hellbender/utils/iterators/ShardedIntervalIterator.java

```diff
@@ -70,11 +70,6 @@ private SimpleInterval calculateShardedInterval() {
     }
 
     @Override
-    public Iterator<SimpleInterval> iterator() {
-        return this;
-    }
-
-    @Override
```

Comment:
Ha! sorry I realize now that you are just getting the override from the other method, so is fine, ok forget this one and the previous too.